conftest.py

```diff
@@ -127,6 +131,55 @@ def pytest_keyboard_interrupt(excinfo):
         exp.stop()
 
 
+@pytest.fixture
+def app_id(request):
```

Comment:
request isn't used in most of the fixtures, so you can just remove them where unused ;-).

Reply:
done